pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sphinx_gherkindoc"
-version = "3.4.0"
+version = "3.4.1"
 description = "A tool to convert Gherkin into Sphinx documentation"
 authors = ["Lewis Franklin <lewis.franklin@gmail.com>", "Doug Philips <dgou@mac.com>"]
```

Comment:
NB: Maybe add yourself as an author? :) You know. If you want.

Reply:
I think there is a whole section of contributors elsewhere in the docs.

Reply:
Good point -- I'll drop my name from the authors list in #33